deployment.yaml

```
@@ -37,3 +37,6 @@ spec:
               port: 3000
             initialDelaySeconds: 5
             periodSeconds: 5
+      imagePullSecrets:
+        - name: acr-secret
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)